### ISSUE TITLE
How can I retrain this model?

### ISSUE THREAD

**Issue by jqly** [subeeshvasu/2018_subeesh_epsr_eccvw#3]
Very cool stuff! However, this repo only contains code for testing. Could you please add some comments on readme about how to retrain this model?

**Comment by subeeshvasu** [comment]
Hello,

One can use the code released by authors of EDSR (https://github.com/thstkdgus35/EDSR-PyTorch) to reproduce the results. Run the code following the format in - (https://github.com/thstkdgus35/EDSR-PyTorch/blob/master/src/demo.sh). We used the following training options, --template GAN,  --gan_k 2, --res_scale 0.1, --patch_size 192, and  --loss p1 x VGG54+p2 x GAN+p3 x MSE. The exact parameter values used in our experiments can be found in our paper (https://arxiv.org/pdf/1811.00344.pdf).

**Comment by jqly** [comment]
Thanks!

**Comment by shazib-summar** [comment]
Are the values of these parameters the same as the one you used for region 1 of PIRM2018? I am asking because these values do not match the default values of options.py in this repo.

**Comment by shazib-summar** [comment]
Also, which pre-trained model file do I supply for weight initialization if I'm retraining EPSR (region 1) from the EDSR repo?

**Comment by subeeshvasu** [comment]
> Are the values of these parameters the same as the one you used for region 1 of PIRM2018? I am asking because these values do not match the default values of options.py in this repo.

Yes. The parameter values of my experiments are mentioned in the paper

**Comment by subeeshvasu** [comment]
> Also, which pre-trained model file do I supply for weight initialization if I'm retraining EPSR (region 1) from the EDSR repo?

I have mentioned it in the paper, "we initialized the model weights of generator using pre-trained weights of EDSR
(obtained by training EDSR with λ1 = λ3 = 0)." You can either use the pretrained weights supplied by authors of EDSR (this should be easy to find out), or else train EDSR with this settings by urself and then use those weights.
